packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -411,51 +415,51 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
   /// would return null. This will allow showing nested data for properties
   /// that don't show children by default in other debugging output but
   /// could.
-  bool get isDiagnosticableValue {
+  bool? get isDiagnosticableValue {
     return getBooleanMember('isDiagnosticableValue', false);
   }
 
-  String getStringMember(String memberName) {
+  String? getStringMember(String memberName) {
     return JsonUtils.getStringMember(json, memberName);
   }
 
   bool getBooleanMember(String memberName, bool defaultValue) {
     if (json[memberName] == null) {
       return defaultValue;
     }
-    return json[memberName];
+
+    return json[memberName] as bool;
   }
 
   DiagnosticLevel getLevelMember(
       String memberName, DiagnosticLevel defaultValue) {
-    final String value = json[memberName];
+    final String? value = json[memberName] as String?;
     if (value == null) {
       return defaultValue;
     }
-    final level = diagnosticLevelUtils.enumEntry(value);
-    assert(level != null, 'Unabled to find level for $value');
-    return level ?? defaultValue;
+    final level = diagnosticLevelUtils.enumEntry(value)!;
+    return level;
   }
 
   DiagnosticsTreeStyle getStyleMember(
       String memberName, DiagnosticsTreeStyle defaultValue) {
     if (!json.containsKey(memberName)) {
       return defaultValue;
     }
-    final String value = json[memberName];
+    final String? value = json[memberName] as String?;
     if (value == null) {
       return defaultValue;
     }
-    final style = treeStyleUtils.enumEntry(value);
-    assert(style != null);
-    return style ?? defaultValue;
+    final style = treeStyleUtils.enumEntry(value)!;
+    return style;
   }
 
   /// Returns a reference to the value the DiagnosticsNode object is describing.
-  InspectorInstanceRef get valueRef => InspectorInstanceRef(json['valueId']);
+  InspectorInstanceRef get valueRef =>
+      InspectorInstanceRef(json['valueId'] as String?);
 
   bool isEnumProperty() {
-    return type != null && type.startsWith('EnumProperty<');
+    return type != null && type!.startsWith('EnumProperty<');
```

Comment:
change to `type?.startsWith('EnumProperty<') ?? false;` instead

Reply:
done